dotcom-rendering/src/lib/article-count.ts

```diff
@@ -0,0 +1,32 @@
+import { storage } from '@guardian/libs';
+import {
+	getWeeklyArticleHistory,
+	incrementWeeklyArticleCount,
+} from '@guardian/support-dotcom-components';
+import { hasOptedOutOfArticleCount } from '../web/lib/contributions';
+import { incrementDailyArticleCount } from '../web/lib/dailyArticleCount';
+
+// Log an article view using the Slot Machine client lib
```

Comment:
What is Slot Machine? I can't see it referenced in the code.

Reply:
this is an old comment - slot machine was an experiment which didn't really go anywhere.
I suggest deleting the comment